packages/controller/jest.config.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2020 Opstrace, Inc.
```

Comment:
New files need to have the copyright set to the current year. Please change to 2021

Reply:
ah yeah, makes sense